assets/dev/js/editor/components/ai/page-content.js

```diff
@@ -0,0 +1,89 @@
+import { useEffect } from 'react';
+import FormText from './pages/form-text';
+import Connect from './pages/connect';
+import FormCode from './pages/form-code';
+import GetStarted from './pages/get-started';
+import Loader from './components/loader';
+import useUserInfo from './hooks/use-user-info';
+import WizardDialog from './components/wizard-dialog';
+import PromptDialog from './components/prompt-dialog';
+import UpgradeChip from './components/upgrade-chip';
+
+const PageContent = (
+	{
+		type,
+		controlType,
+		onClose,
+		getControlValue,
+		setControlValue,
+		additionalOptions,
+	} ) => {
+	const { isLoading, isConnected, isGetStarted, connectUrl, fetchData, hasSubscription, credits } = useUserInfo();
+
+	useEffect( () => {
+		fetchData();
+	}, [] );
+
+	if ( isLoading ) {
+		return (
+			<PromptDialog onClose={ onClose }>
+				<Loader />
+			</PromptDialog>
+		);
+	}
+
+	if ( ! isConnected ) {
+		return (
+			<WizardDialog onClose={ onClose }>
+				<Connect connectUrl={ connectUrl } onSuccess={ fetchData } />
+			</WizardDialog>
+		);
+	}
+
+	if ( ! isGetStarted ) {
+		return (
+			<WizardDialog onClose={ onClose }>
+				<GetStarted onSuccess={ fetchData } />
+			</WizardDialog>
+		);
+	}
```

Comment:
```suggestion
	if ( isGetStarted ) {
		return (
			<WizardDialog onClose={ onClose }>
				<GetStarted onSuccess={ fetchData } />
			</WizardDialog>
		);
	}
```

shouldn't be like this?

Reply:
The logic is OK.